source/features/highlight-collaborators-prs.tsx

```diff
@@ -0,0 +1,46 @@
+import './highlight-collaborators-prs.css';
+import select from 'select-dom';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import {getRepoGQL} from '../libs/utils';
+
+function buildQuery(): string {
+	return `
+		repository(${getRepoGQL()}) {
+			collaborators(first: 100) {
+				nodes { login }
+			}
+		}
+	`;
+}
+
+async function init(): Promise<false | void> {
+	if (!features.isRepoWithAccess()) {
```

Comment:
There's an `exclude` array in `features.add`

Reply:
> There's an exclude array in features.add

:+1:

> But is this needed?

Yeah, this is needed because it only works on repos which you have settings rights to. On this repo I get the following error:
```
{
  "data": {
    "repository": {
      "collaborators": null
    }
  },
  "errors": [
    {
      "type": "FORBIDDEN",
      "path": [
        "repository",
        "collaborators"
      ],
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "message": "Must have push access to view repository collaborators."
    }
  ]
}
```

> Also it should probably be renamed to canUserCollaborate or something more specific.

That can be in another PR.

Reply:
> Yeah, this is needed because it only works on repos which you have settings rights to. On this repo I get the following error:

Wow, what an ugly limitation. I think this limits its usefulness a bit. Do you think it's worth using this hack? https://stackoverflow.com/a/40987268/288906
Parsing the DOM here is not different from what we do across the extension (e.g. `forked-to`)

Reply:
Interesting. The _hack_ is already available for almost 3 year, it might be _safe_ to use.

But I'm not sure if we should go that way. This [feature request](https://github.com/sindresorhus/refined-github/issues/438#issue-231104704) was only about viewing PRs from other collaborators (who should have push access to the repo). And as owner I'm most interested in this information.

Aldo this information might be interesting for other 'normal' users too. 🤔

I could add the url hack as a fallback in case the API v4 throws...

Reply:
I also noticed that the url hack included my own name, which is false in the case of the [example](https://github.com/Leaflet/Leaflet/issues/show_menu_content?partial=issues/filters/authors_content).

Reply:
> I could add the url hack as a fallback in case the API v4 throws...

No, there should only be one thing. We can't maintain 2.

Reply:
> I also noticed that the url hack included my own name, which is false in the case of the [example](https://github.com/Leaflet/Leaflet/issues/show_menu_content?partial=issues/filters/authors_content).

Indeed, but that's your own name, it doesn't need this highlight (and it already is)

Reply:
So, you want me to change the implementation to the **url hack**, instead of the **limited API v4** ?

Reply:
I think it's best to go with the URL hack so it becomes more useful.